From e2b6fbe88ab4d98a7960281ac7568dbd6e7927b4 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 27 Dec 2017 20:00:14 +0200
Subject: [PATCH 156/249] lineage: Reenable AudioFX and remove
 LineageAudioService
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

* Lets have some ＥＦＦＥＣＴＳ
* LineageAudioService is integrated into Lineage SDK again

Change-Id: I7ef62bc32edc8e575c5d57c23532f12757c8d44d
---
 snippets/lineage.xml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index 1450bcb..4618032 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -35,7 +35,7 @@
   <!--<project path="lineage-sdk/samples/weatherproviderservice/OpenWeatherMapProvider" name="LineageOS/android_packages_apps_OpenWeatherMapProvider" />-->
   <!--<project path="lineage-sdk/samples/weatherproviderservice/WundergroundWeatherProvider" name="LineageOS/android_packages_apps_WundergroundWeatherProvider" />-->
   <!--<project path="lineage-sdk/samples/weatherproviderservice/YahooWeatherProvider" name="LineageOS/android_packages_apps_YahooWeatherProvider" />-->
-  <!--<project path="packages/apps/AudioFX" name="LineageOS/android_packages_apps_AudioFX" />-->
+  <project path="packages/apps/AudioFX" name="LineageOS/android_packages_apps_AudioFX" />
   <project path="packages/apps/Eleven" name="LineageOS/android_packages_apps_Eleven" />
   <!--<project path="packages/apps/Exchange" name="LineageOS/android_packages_apps_Exchange" />-->
   <project path="packages/apps/FMRadio" name="LineageOS/android_packages_apps_FMRadio" />
@@ -50,7 +50,6 @@
   <!--<project path="packages/apps/Updater" name="LineageOS/android_packages_apps_Updater" />-->
   <!--<project path="packages/providers/DataUsageProvider" name="LineageOS/android_packages_providers_DataUsageProvider" />-->
   <!--<project path="packages/providers/WeatherProvider" name="LineageOS/android_packages_providers_WeatherProvider" />-->
-  <!--<project path="packages/services/LineageAudioService" name="LineageOS/android_packages_services_LineageAudioService" />-->
   <!--<project path="packages/services/WeatherService" name="LineageOS/android_packages_services_WeatherService" />-->
   <!--<project path="prebuilts/lineage-sdk" name="LineageOS/android_prebuilts_lineage-sdk" clone-depth="1" />-->
   <!--<project path="system/extras/su" name="LineageOS/android_system_extras_su" />-->
-- 
2.17.1

